app/views/repositories/_share_repository_modal.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="modal" id="share-repo-modal" tabindex="-1" role="dialog" >
+  <%= form_for :shares, url: multiple_update_team_repository_team_repositories_path(current_team, @repository), remote: :true do |f| %>
+    <div class="modal-dialog" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+          <h4 class="modal-title" id="create-repo-modal-label">
+            <%= t("repositories.index.modal_share.title") %>
+          </h4>
+        </div>
+        <div class="modal-body">
+          <div>
+            <table style="width: 80%; margin: 0 auto">
+              <tr>
+                <th>Share with Team</th><th>Can Edit</th>
```

Comment:
:eyes:

Reply:
We have a ticket for update FE/design of this modal :)